_data/sw_clients.yml

```diff
@@ -924,6 +924,26 @@
           - plain
           - external
           - scram-sha-256
+    - name: Eggdrop
+      link: https://github.com/eggheads/eggdrop
```

Comment:
Can you provide any reference link in source?

Reply:
Sure, sorry, I wasn't sure of the intent of that line- what is this intended to reference? If code, it's kinda smashed all throughout. Maybe a reference to the config file?

Reply:
You can link to the list of capabilities it requests, and add an extra link for the few extensions with no capabilities